samples/c/hello_classification/main.c

```diff
@@ -8,7 +8,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "c_api/ov_c_api.h"
+#include "openvino.h"
```

Comment:
Can we have `openvino/openvino.h`?

Reply:
done

Reply:
Please use always path `<openvino/c/openvino.h>`

Reply:
It will bring compilation error due to the different directory between source code and reference path

source code: src/bindings/c/include/openvino/openvino.h
install directory:  runtime/include/openvino/c/openvino.h

Actually, I'm very confused the rule you set C header file path.
Could you give a final and reasonable solution about it?

Reply:
@riverlijunjie I propose to align paths which we use for C API in the source code and in the samples.
So it means if we use `openvino/openvino.h` in our samples, we should use the same path in our code.

The initial proposal was to have `openvino/openvino.h` but in this case we need to install c API in the separate folder, in order to avoid mix C and C++ API.

Moreover I start to think that `openvino/c/openvino.h` path has some disadvantages for our package, because in this case we mix C++ and C API, and if user just use C++ openvino, he will be able to include C API, it is not correct because openvino library doesn't contain C API.

We need to have a separate folder for C API in order to avoid intersection with C++ headers and in order to have different include paths for C++ and C.

Reply:
So at the current moment let's align all paths to `openvino/c/openvino.h` it should work as in sample application and at the same moment it should work inside openvino.

I will think about the possible mix of APIs, and will return in the nearest time with final proposal about the path, because at the current moment it is very controversial to have C API inside folders of C++ API, but we had it before for C API and IE API.

Reply:
done